packages/turbo/ios/RNVisitableView.swift

```diff
@@ -10,22 +10,53 @@ import UIKit
 
 class RNVisitableView: UIView, SessionSubscriber {
   var id: UUID = UUID()
+  @objc var sessionHandle: NSString? = nil {
+    didSet {
+      validateSessionProperty(propertyName: "sessionHandle", oldValue: oldValue, newValue: sessionHandle)
+    }
+  }
+  @objc var applicationNameForUserAgent: NSString? = nil {
+    didSet {
+      validateSessionProperty(propertyName: "applicationNameForUserAgent", oldValue: oldValue, newValue: applicationNameForUserAgent)
+    }
+  }
```

Comment:
I think the property validation here are a bit too clever for now. I don't imagine that consuming developers would change these values dynamically and we add complexity here for very little gain.